File: frontend/src/types/dto/area.dto.ts
export interface AreaDTO {
  areaId: string;
  branchId: string;
  name: string;
  description?: string;
  createdAt?: string;
  updatedAt?: string;
}

export interface AreaCreateRequest {
  branchId: string;
  name: string;
  description?: string;
}

File: backend/src/main/java/com/example/backend/dto/AreaDTO.java
package com.example.backend.dto;

import lombok.*;
import java.time.Instant;
import java.util.UUID;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class AreaDTO {

    private UUID areaId;

    private UUID branchId;

    private String name;

    private String description;

    private Instant createdAt;

    private Instant updatedAt;

}
